#! /bin/sh -e

# DP: compile capidyn.c using -fPIC

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p0 < $0
        #cd ${dir}gcc && autoconf
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p0 < $0
        #rm ${dir}gcc/configure
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

--- capi20/Makefile.am~	2004-06-14 13:23:47.000000000 +0200
+++ capi20/Makefile.am	2004-08-09 07:42:40.000000000 +0200
@@ -15,6 +15,7 @@
 libcapi20dyn_a_SOURCES = capidyn.c
 libcapi20dyn_a_CFLAGS = -fPIC
 # libcapi20_la_LIBADD = $(LIBADD_DL)
+CFLAGS = -fPIC -DPIC -g -O2
 
 $(OBJECTS): libtool
 
